modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/auto_fields.js

```diff
@@ -419,20 +425,20 @@ AUI.add(
 
 								item.attr('checked', '');
 								item.attr('name', newName);
-								item.attr('id', newName);
+								item.attr('id', newId);
 							}
 							else if (
 								inputNodeName === 'button' ||
 								inputNodeName === 'div' ||
 								inputNodeName === 'span'
 							) {
 								if (oldName) {
-									item.attr('id', newName);
+									item.attr('id', newId);
 								}
 							}
 							else {
 								item.attr('name', newName);
-								item.attr('id', newName);
+								item.attr('id', newId);
 							}
 
 							if (fieldStrings && fieldStrings[oldName]) {
```

Comment:
is this affected by the new way of computing `oldName` ?  I ask because if there are both `id` and `name` , with your changes, we'll be looking at a different entry in `fieldStrings` and perhaps this is not desired. 

Same consideration wouild apply to `rules` and `aria-describedby`, a bit below.

Reply:
The new way of computing `oldName` is actually the classic way to do it until we changed it two weeks ago under LPD-53924, so this should not be affected.